apps/mull-ui/src/constants.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Map of the current routes for the application
+ */
+export const ROUTES = {
+  HOME: '/home',
+  MAP: '/map',
+  NEW_EVENT: '/new-event',
```

Comment:
Not blocking, but to match with how we named our EPICs/USs/requirements, I would change this name to `/create-event`

Reply:
Sure thing